components/autosize_textarea.jsx

```diff
@@ -116,14 +123,19 @@ export default class AutosizeTextarea extends React.Component {
         }
 
         return (
-            <div>
+            <div className=''>
+                <div
+                    ref='placeholder'
+                    className='custom-textarea__placeholder hidden'
```

Comment:
Also, instead of adding the `custom-textarea__placeholder` here, is there any way that we can have the placeholder use the same className as the other textareas, plus some inline styles to make it look like a placeholder? The problem with doing it this way is that this component is supposed to be usable outside of the `custom-textarea`, so we shouldn't be applying styles to it that are specific to the `custom-textarea`. I'm thinking something like
```
<div
    className={this.props.className} 
    style={style.placeholder}
>
    {placeholder}
</div>

// At the bottom of the file with the other inline styles
const style = {
    placeholder: {
        overflow: 'hidden',
        textOverflow: 'ellipsis',
        opacity: 0.5,
    },
};
```